docker/metrics/docker-compose.yml

```
@@ -0,0 +1,38 @@
+version: "3.9"
+
+services:
+  grafana:
+    restart: $RESTART_POLICY
+    image: grafana/grafana-oss
+    volumes:
+      - grafana-storage:/var/lib/grafana
+    ports:
+      - "${GRAFANA_EXPOSE:-3000}:3000"
```

here also, I would prefer default to be on 127.0.0.1 (security by default)

```suggestion
      - "${GRAFANA_EXPOSE:-127.0.0.1:3000}:3000"
```